src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.targets

```diff
@@ -44,6 +44,15 @@
 		<_UnoProjectSystemPackageReference Include="Uno.Extensions.Markup.Generators" ProjectSystem="true" />
 	</ItemGroup>
 
+	<ItemGroup Condition="$(UnoFeatures.Contains(';mauiembedding;'))">
```

Comment:
It feels like this section is not required, it's a duplicate of the other section in uno.extensions. It's likely, though, that the bug really was the typo that you found!

Reply:
Done, it should work now